framework/components/ecl-fields/ecl-fields.config.js

```diff
@@ -1,17 +1,33 @@
 module.exports = {
   title: 'Fields',
   label: 'Fields',
-  status: 'planned',
-  collated: false,
+  status: 'ready',
+  collated: true,
+  collator(markup, item) {
+    return `
+      <!-- Start: @${item.handle} -->\n
+      <h3 class="ecl-heading ecl-heading--h3">${item.label}\n</h3>\n
+      ${markup}\n
+      <!-- End: @${item.handle} -->\n
+    `;
+  },
   tags: ['atom'],
   variants: [
     {
       name: 'default',
       label: 'Default',
+      context: {
+        label: 'Name',
+        body: 'John Doe',
```

Comment:
I think they will ask about the examples with icons, links and non-textual content they demo in current implementation.